src/lib/ui/thermostat/ClimateSettingScheduleDevice.tsx

```diff
@@ -0,0 +1,54 @@
+import { isThermostatDevice, type ThermostatDevice } from 'seamapi'
+
+import { ChevronRightIcon } from 'lib/icons/ChevronRight.js'
+import { useDevice } from 'lib/seam/devices/use-device.js'
+import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+export function ClimateSettingDevice({
```

Comment:
What is a `ClimateSettingScheduleDevice`? Is it different than a thermostat?

Reply:
Fair point, this was just to keep it seperate. Something with `DeviceBar` would be better.

Reply:
It's possible these components you're adding actually don't belong in UI. They look more like the sub components that will exist when we implement the top level component. Take a look at the other folders in lib/seam to see what I'm referring to.